src/mc.rs

```diff
@@ -1920,14 +1929,16 @@ pub(crate) unsafe extern "C" fn warp_affine_8x8_c_erased<BD: BitDepth>(
 // TODO(legare): Temporarily pub until init fns are deduplicated.
 pub(crate) unsafe extern "C" fn warp_affine_8x8t_c_erased<BD: BitDepth>(
     tmp: *mut i16,
-    tmp_stride: ptrdiff_t,
+    tmp_stride: usize,
     src: *const DynPixel,
     src_stride: ptrdiff_t,
     abcd: &[i16; 4],
     mx: c_int,
     my: c_int,
     bitdepth_max: c_int,
+    tmp_len: usize,
 ) {
+    let tmp = std::slice::from_raw_parts_mut(tmp, tmp_len);
```

Comment:
Also tiny little nit:

```suggestion
    let tmp = slice::from_raw_parts_mut(tmp, tmp_len);
```

Reply:
All the other instances of this in the file are doing `std::slice::from_raw_parts`, so I'm going to import `std::slice` and update all the instances to `slice::from_raw_parts`.